espwrap/base.py

```diff
@@ -58,99 +58,52 @@ def add_recipient(self, email, name='', merge_vars={}):
         # tuple for performance
         self.recipients = itertools.chain(self.recipients, (recip,))
 
-        return self
-
-    def add_recipients(self, recipients):
+    def add_recipients(self, *recipients):
         for recip in recipients:
             self.add_recipient(recip)
 
-        return self
-
     def clear_recipients(self):
         self.recipients = list()
 
-        return self
-
-    def add_global_merge_var(self, key, value, overwrite=False):
-        if not overwrite and self.global_merge_vars.get(key):
-            raise Exception(
```

Comment:
Lost functionality but this function is pretty explicit so I guess it's not the "python way" to shield the user from this